main.go

```diff
@@ -34,10 +35,19 @@ func main() {
 		d.Init(discordToken)
 	}
 
+	dbArgs := database.DbInfo{
+		Host: env.GetEnv("POSTGRES_HOST"),
```

Comment:
Let's update the README to include these new postgres env vars

Reply:
This shouldn't need to be defined in the .env file since the database we're using is also running in the same docker container. These are hardcoded in `docker-compose.yml`

Reply:
Ok that sounds good to me